tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_Contacts_RTL.py

```diff
@@ -0,0 +1,42 @@
+import pdb
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.contacts.app import Contacts
+from gaiatest.apps.contacts.regions.settings_form import SettingsForm
+from gaiatest.apps.contacts.regions.contact_form import ContactForm
```

Comment:
You don't need to import the 2 above imports if you follow what I suggested below.